src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.debug("login method was called. Performing login for={}", login);
```

Comment:
minor

on real environments logs are gathered and stored in some files on some machines

this implies some memory usage, since these files are usually large (20 GB, 30 GB etc.)

this means that you should try to keep your log messages as short as possible but at the same time they should be self-explanatory to bring some value

considering that I would suggest
```suggestion
        logger.debug("Performing login for={}", login);
```